src/main.js

```diff
@@ -4,4 +4,14 @@ import App from './App.vue'
 import './styles/common/index.scss'
 import './styles/global/index.scss'
 
-createApp(App).mount('#app')
+import Datepicker from '@vuepic/vue-datepicker';
+import '@vuepic/vue-datepicker/dist/main.css';
+
+import router from'@/router';
+
+const app = createApp(App);
+
+app.use(router)
+app.component('Datepicker', Datepicker);
```

Comment:
I would suggest to not register such component globally. It's not something you are going to commonly use across the application - import it where you need it instead.

Reply:
Good point! It was probably added automatically during installation and I haven't notice it.